index.html

```diff
@@ -88,22 +91,34 @@ <h1 class="brand-heading">Brain&nbsp;Imaging Data&nbsp;Structure</h1>
+        <p>A good introduction to the BIDS standard can be found in the <a href="http://www.nature.com/articles/sdata201644">paper published in Nature Scientific Data</a>.</p>
+            </div>
+        </div>
+    </section>
+
+    <section id="benefits" class="container content-section text-left">
+        <div class="row">
+            <div class="col-lg-12">
+                <h2>Benefits</h2>
+        <p>By using this standard you will benefit in the following ways:
         <ul>
           <li>It will be easy for another researcher to work on your data. To understand the organization of the files and their format you will only need to refer them to this document. This is especially important if you are running your own lab and anticipate more than one person working on the same data over time. By using BIDS you will save time trying to understand and reuse data acquired by a graduate student or postdoc that has already left the lab.</li>
           <li>There is a growing number of data analysis software packages that can understand data organized according to BIDS.</li>
           <li>Databases such as <a href="http://openfmri.org">OpenfMRI.org</a>, <a href="http://www.loris.ca">LORIS</a>, <a href="https://portal.mrn.org">COINS</a>, <a href="https://central.xnat.org/">XNAT</a>, <a href="https://scitran.github.io/">SciTran</a> and others will accept and export datasets organized according to BIDS. If you ever plan to share your data publicly (nowadays some journals require this) you can speed up the curation process by using BIDS.</li>
           <li>There are <a href="https://github.com/Squishymedia/bids-validator">validation tools</a> (also available <a href="http://incf.github.io/bids-validator/">online</a>) that can check your dataset integrity and let you easily spot missing values.</li>
           <li>We provide <a href="https://github.com/INCF/openfmri2bids">tools to convert OpenfMRI style organized data to BIDS</a></li>
         </ul></p>
-        <p>BIDS is heavily inspired by the format used internally by <a href="http://openfmri.org">OpenfMRI.org</a>. While working on BIDS we consulted many neuroscientists to make sure it covers most common experiments, but at the same time is intuitive and easy to adopt. The specification is intentionally based simple file formats and folder structures to reflect current lab practices and make it accessible to wide range of scientists coming from different backgrounds.</p>
-        <p>Software currently supporting BIDS:</p>
+        <p id="software">Software currently supporting BIDS:</p>
```

Comment:
Did you intend to use a link to this id? If so I cannot see it in the diff.

Reply:
Well, I was intending to use it "from outside", i.e. whenever I want to refer to it, but did not have intent to blow up the menu. BUT now I realized that I could refer to it from above `<li>`, will do now